cross/gdb-latest/Makefile

```diff
@@ -19,10 +19,12 @@ UNSUPPORTED_ARCHS = $(ARMv5_ARCHS) $(PPC_ARCHS)
 
 include ../../mk/spksrc.cross-cc.mk
 
-# Library is not created under armv7
-ifeq ($(findstring $(ARCH),$(ARMv7_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
+# Library is not created for armv5 and armv7
+ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS) $(ARMv7_ARCHS) $(ARMv7L_ARCHS)),$(ARCH))
 PLIST_TRANSFORM = sed -e 's?lib:lib/libinproctrace.so??g'
-# binary only created on armv7
-else
+endif
```

Comment:
Also missing is :
```
# disable simulator ; it's broken on ppc
# https://scm.linefinity.com/common/openwrt/commit/079d57b0f290a79c9dbc013b6e9c83cebf8a2f99
ifeq ($(findstring $(ARCH),$(PPC_ARCHS)),$(ARCH))
CONFIGURE_ARGS += --disable-sim
endif
```

Further: https://sourceware.org/pipermail/gdb/2023-September/050898.html

Reply:
Also, this applies to both gdb-7.12.1 and gdb-latest

Reply:
That was my finding: gdb-latest (13.2) has broken sim for qoriq, but gdb-7.12 has not.
It builds fine with the provided patches.

Reply:
see my comment https://github.com/SynoCommunity/spksrc/pull/5879#issuecomment-1717396995

Reply:
From my readings sim isn't needed anyway.  So while it now build for 7.12.1 I still recommend we deflect that and use gdb-latest wth --disable-sim.

Also, more I'm looking at how it's handling the compiler from within `sim/ppc` more I doubt it will ever work correctly due to using "build" compiler instead of "target".  I'd go further and recomment using --disable-sim on both versions for qoriq and use gdb-latest.

Further adding, the test for switching between 7.12 and 3.2 should be if gcc <= 4.9 or similar instead of being armv5 specific.

Reply:
> From my readings sim isn't needed anyway. So while it now build for 7.12.1 I still recommend we deflect that and use gdb-latest wth --disable-sim.

So I will disable sim for all archs?

Reply:
> Further adding, the test for switching between 7.12 and 3.2 should be if gcc <= 4.9 or similar instead of being armv5 specific.

_GCC 4.8.1 was the first feature-complete implementation of the 2011 C++ standard..._

So I will check for gcc < 4.8.1 (not <= 4.9)

Reply:
> > From my readings sim isn't needed anyway. So while it now build for 7.12.1 I still recommend we deflect that and use gdb-latest wth --disable-sim.
> 
> So I will disable sim for all archs?

No, I'd disable it only for qoriq as it works ok for all others.  Unless we want to go through an optimization of gdb and disable everything that is uneeded.